File: src/main/java/de/rubixdev/enchanted_shulkers/asm/PortableContainerTarget.java
package de.rubixdev.enchanted_shulkers.asm;

import de.rubixdev.enchanted_shulkers.Utils;
import de.rubixdev.enchanted_shulkers.mixin.EnchantmentTargetMixin;
import net.minecraft.item.Item;

@SuppressWarnings("unused") // used in de.rubixdev.enchanted_shulkers.asm.EnumInjector
public class PortableContainerTarget extends EnchantmentTargetMixin {
    @Override
    public boolean isAcceptableItem(Item item) {
        return Utils.canEnchant(item);
    }
}

File: src/main/kotlin/de/rubixdev/enchantedshulkers/asm/PortableContainerTarget.kt
package de.rubixdev.enchantedshulkers.asm

import de.rubixdev.enchantedshulkers.Utils
import net.minecraft.item.Item

class PortableContainerTarget : EnchantmentTargetMixin() {
    override fun isAcceptableItem(item: Item?) = Utils.canEnchant(item)
}
